code/model/DMSDocumentSet.php

```diff
@@ -231,7 +239,23 @@ public function addQueryFields($fields)
         ));
 
         $sortedBy->setTitle(_t('DMSDocumentSet.SORTED_BY', 'Sort the document set by:'));
-        $fields->addFieldsToTab('Root.QueryBuilder', array($keyValPairs, $sortedBy));
+        $fields->addFieldsToTab(
+            'Root.QueryBuilder',
+            array(
+                LiteralField::create(
+                    'GridFieldNotice',
+                    '<p class="message warning">' . _t(
+                        'DMSDocumentSet.QUERY_BUILDER_NOTICE',
+                        'The query builder provides the ability to add documents to a ' .
+                        'document set based on the filters below. Please note that the set will be built using this '.
+                        'criteria when you save the form. It will not dynamically update. [See the docs for an ' .
+                        'example of how this could be run on a scheduled job.]'
```

Comment:
Maybe we could just say "See the documentation for more information." and leave the scheduled job etc to the docs to explain?